images/win/scripts/Installers/Validate-Boost.ps1

```diff
@@ -55,6 +55,10 @@ foreach ($BoostTool in $BoostTools)
 $CMakeFindBoostInfo = @"
 
 #### _Notes:_
+
+> ``BOOST_PATH`` was deprecated on images but it is required by CMake. Please make sure you set this variable
```

Comment:
I don't think that we should write `was deprecated` in documentation. it is more for changelog :)
I would consider something like `BOOST_ROOT` environment variable is not set up on the images.

Reply:
thanks, updated